tests/integration/test_clip.py

```diff
@@ -8,14 +8,13 @@
 from PIL import Image
 from jina import Flow, Document, DocumentArray
 
-from encoder_clip import CLIPImageEncoder
+from jinahub.encoder.clip_image import CLIPImageEncoder
 
 cur_dir = os.path.dirname(os.path.abspath(__file__))
 
 
 def test_clip_any_image_shape():
     def validate_callback(resp):
-        print(DocumentArray(resp.data.docs).get_attributes('embedding'), 'embedding')
         assert 1 == len(DocumentArray(resp.data.docs).get_attributes('embedding'))
```

Comment:
U can directly do `resp.docs.get_attributes('embedding')`

Reply:
At least in new versions

Reply:
thanks, done